gravatar/src/main/java/com/gravatar/services/ProfileService.kt

```diff
@@ -51,7 +58,7 @@ public class ProfileService(okHttpClient: OkHttpClient? = null) {
      * @param email The email address to fetch the profile for
      * @return The fetched profiles
      */
-    public suspend fun fetch(email: Email): Result<Profile, ErrorType> {
+    public suspend fun fetch(email: Email): Result<LegacyProfile, ErrorType> {
```

Comment:
Shouldn't we deprecate all the `fetch` functions?

Reply:
Good catch! Fixed!